sass/_b3-support-reviews.scss

```diff
@@ -0,0 +1,120 @@
+.b3-support-reviews{
+  background: $background-B;
+  color: $white;
+  padding-bottom: 1em;
+  padding-top: 1em;
+}
+.reviews-container{
+  margin: auto;
+}
+.b3-support-reviews {
+ h1 {
+  font-family: "Raleway";
+  font-weight: 300;
+  font-size: 3em;
+ }
+ h2, p {
+  font-family: "Roboto";
+  font-style: lighter;
+  font-size: 1em;
+  }
+}
+.icon-title{
+  display: flex;
+  -webkit-display: flex;
+  justify-content: center;
+  -webkit-justify-content: center;
+}
+.container-icon-review{
+  background-color: transparent;
+  height: 100px;
+  width: 100px;
+  border-top: 8px solid transparent;
+  border-bottom: 5px solid $white;
+  border-left: 1px solid transparent;
+  border-right: 1px solid transparent;
+  border-radius: 50%;
+  margin: auto;
+  display: flex;
+  span {
+    align-items: center;
+    color: $white;
+    font-size: 2em;
+  }
+}
+.arrow-align {
+  display:flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.carousel-outer-reviews {
+  width: 900px;
+  height: 250px;
+  display: block;
+  position: relative;
+  margin: auto;
+}
+.carousel-inner-reviews{
+  width: 900px;
+  height: 250px;
+  display: block;
+  position: absolute;
+  overflow: hidden;
+  margin: auto;
+}
+.review-text-display {
+  padding-top: 5%;
+  width: 900px;
+  height: 250px;
+  background: $background-B;
+    p {
+      position: relative;
+      display: inline;
+      font-size: 16px;
+    }
+}
+.carousel-inner-reviews {
+  display:flex;
```

Comment:
Please be consistent with your spacing; either `display: flex` here OR don't add a space after the colon for your other properties.